### ISSUE TITLE
Is the xml/rss output suitable for iTunes?

### ISSUE THREAD

**Issue by saigelosli** [maxnowack/node-podcast#48]
I am using version 2.0.0 of podcast npm module. I know that I can index my podcast through iTunes, and users can then subscribe _through_ iTunes. But I was hoping that providing this xml file on my web page would allow users of iOS devices to simply tap on the link, and the XML contained herein would allow their Podcasting app to subscribe to my podcast without going through iTunes. Instead, they get:

"Cannot Open Page
This is a link to an RSS feed. Would you like to search the App Store for apps that can display RSS feeds?"

For clarity, I am offering the XML output from my server code like this:

`app.get( "/podcast/weeklyReading/feed.rss", ( req, res, next ) => {
    res.header( "Content-Type", "application/rss+xml" )
    res.status( 200 ).send( outputXML )
})`

It _does_ provide the XML output, but that is apparently not sufficient to get the Podcast app to subscribe without copying and pasting the "/podcast/weeklyReading/feed.rss" URL into the Podcast app.

If I am completely off track, I would appreciate a pointer to documentation that will help be get back on track.

**Comment by bleeckerj** [comment]
Hi @saigelosli — did you ever find out what the issue was with this problem?

**Comment by maxnowack** [comment]
I don't think that this is an issue with this library. The podlove subscribe button maybe is a solution: https://podlove.org/podlove-subscribe-button/
